packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -357,6 +357,38 @@
         "title": "%force_lightning_lwc_test_stop_watching_all_tests_text%"
       }
     ],
+    "debuggers": [
+      {
+        "type": "vscode-lwc-tests",
+        "label": "%force_lightning_lwc_debugger%",
+        "languages": [
+          "javascript"
+        ],
+        "configurationSnippets": [
+          {
+            "label": "%force_lightning_lwc_debugger%",
+            "description": "%force_lightning_lwc_debugger_desc%",
+            "body": {
+              "name": "%force_lightning_lwc_debugger%",
+              "type": "node",
+              "request": "launch",
+              "program": "^\"\\${workspaceFolder}/node_modules/.bin/sfdx-lwc-jest\"",
```

Comment:
Will forward slashes for windows as well?

Reply:
I believe so but intend to verify.  I did some splunking at https://github.com/jest-community/vscode-jest/blob/e3dbd7299a13730b34697601a10e323ba6286649/package.json#L337 and followed their existing pattern.

Reply:
@gbockus-sf - I'm the QA for this story as well. I will run a sanity check on windows today.

Reply:
I've been fighting it this AM. I think there will be an update required for windows.  Will keep you posted

Reply:
@jag-sfdc verified with circle build 31400 we are good on windows 👍